### DISCUSSION TITLE
TLS Handshake failed/key negotiation failed IPv6

### DISCUSSION THREAD

**Discussion by thelost404** [angristan/openvpn-install#1113] [Help]
Im new to this OpenVPN stuff. My problem is that im behind NAT and i can only access WAN/internet via IPv6. If i connect via ipv4 everything works (internaly). For configuration i followed the script and some guide. If you need more info please ask.

**Comment by randshell** [comment]
The script supports configuring an IPv6 address, did you answer the questions correctly? Also make sure to use your public reachable IPv6 address to connect.

Posting server and client logs would help. You can replace the sensitive information.

**Comment by thelost404** [comment]
> 
> 
> The script supports configuring an IPv6 address, did you answer the questions correctly? Also make sure to use your public reachable IPv6 address to connect.
> 
> Posting server and client logs would help. You can replace the sensitive information.

Yes i think i answered the question correctly. Im also able to ping the IPv6. 
What logs would you need?
